test/ssltest_old.c

```diff
@@ -1891,7 +1891,7 @@ int doit_localhost(SSL *s_ssl, SSL *c_ssl, int family, long count,
          */
 
         /*
-         * We have non-blocking behaviour throughout this test program, but
+         * We have non-blocking behavior throughout this test program, but
```

Comment:
This makes even less sense :-) If you acknowledge original author's authority on choosing the commentary spelling, then you shouldn't have commented on this change. Or said "don't tell me that comments have to be in British" :-) No? Anyway, I'm ready to plus-one if this modification is reverted. With rationale that original spelling was there first.